components/apimgt/org.wso2.carbon.apimgt.rest.api.endpoint.registry/src/test/java/org.wso2.carbon.apimgt.rest.api.endpoint.registry/impl/RegistriesApiServiceImplTest.java

```diff
@@ -263,23 +228,12 @@ public void addRegistry_ResourceNameExists() throws APIManagementException {
 
     @Test
     public void updateRegistry() throws APIManagementException {
-        EndpointRegistryInfo endpointRegistryInfoOld = new EndpointRegistryInfo();
-        endpointRegistryInfoOld.setName("Endpoint Registry 1");
-        endpointRegistryInfoOld.setUuid("abc1");
-        endpointRegistryInfoOld.setMode(RegistryDTO.ModeEnum.READWRITE.toString());
-        endpointRegistryInfoOld.setType(RegistryDTO.TypeEnum.ETCD.toString());
-        endpointRegistryInfoOld.setOwner(ADMIN_USERNAME);
-
-        RegistryDTO payloadDTO = new RegistryDTO();
-        payloadDTO.setName("Endpoint Registry 2");
-        payloadDTO.setId("abc1");
-        payloadDTO.setMode(RegistryDTO.ModeEnum.READONLY);
-        payloadDTO.setType(RegistryDTO.TypeEnum.WSO2);
-        payloadDTO.setOwner(ADMIN_USERNAME);
-
-        EndpointRegistryInfo endpointRegistryInfoNew =
-                EndpointRegistryMappingUtils.fromDTOtoEndpointRegistry(payloadDTO, ADMIN_USERNAME);
-        endpointRegistryInfoNew.setUuid("abc1");
+        EndpointRegistryInfo endpointRegistryInfoOld = createRegistryWithDefaultParams();
+        EndpointRegistryInfo endpointRegistryInfoNew = createRegistry(endpointRegistryInfoOld.getUuid(), 2,
+                "Endpoint Registry 2", RegistryDTO.ModeEnum.READWRITE, RegistryDTO.TypeEnum.ETCD,
+                "user1");
```

Comment:
Shall we move "user1" to line 233?

Reply:
Removed the file in https://github.com/wso2/carbon-apimgt/pull/8615